bbc/simorgh
# Updates to the nav bars of Afrique, Indonesia, Pashto and Swahili

Resolves JIRA n/a

Overall changes
======
- Removed election topics Afrique and Indoneisa nav bar
- Replaced links to CPS feature indexes with the links to the relevant redirected Tipo topics to avoid an unnecessary redirect on the nav bars of Afrique, Pashto and Swahili
- Fixed Tipo on the last item of Pashto's nav bar

Code changes
======

- Edited Navigation section of src/app/lib/config/services/afrique.ts to remove Election link and replace CPS FIX link
- Edited Navigation section of src/app/lib/config/services/indonesia.ts to remove Election link 
- Edited Navigation section of src/app/lib/config/services/pashto.ts to replace two FIXs links and change the link text of the last item 
- Edited Navigation section of src/app/lib/config/services/swahili.ts to replace CPS FIX link
- Updated snapshots

Testing
======
1. Open Afrique's front page, the second link in the nav should be "Afrique", not Senegal Presiential elections. The Nos emissions link should point to /afrique/topics/c88nzggm8gxt
2. Open Indonesia's front page. There second linkn in the nav should be Indonesia, no longer Pemilu 2024
3. Open Pashto's front page, the eight and ninth links in the nav bar should point to Tipo topics not CPS Fixs
4. Open Swahili's front page. The last link in the nav "Vipindi vya Redio" should point to a Tipo topic not a CPS Fix

Helpful Links
======
_Add Links to useful resources related to this PR if applicable._

[Coding Standards](https://github.com/bbc/simorgh/blob/latest/docs/Coding-Standards/README.md)

[Repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)
